src/datasource.ts

```diff
@@ -315,6 +323,26 @@ export default class CogniteDatasource {
         };
         if (target.tab === Tab.Custom && target.assetQuery.func) {
           query.function = target.assetQuery.func.replace(/ID/g, String(ts.id));
+          const regexSearch = /\[.*?\]/g;
+          const regexMatches = query.function.match(regexSearch);
+          query.aliases = [];
+          for (const match of regexMatches) {
+            // format is id, aggregation, granularity
+            const aliasParts = match
+              .substr(1, match.length - 2)
+              .split(',')
+              .map(x => _.trim(x, ' \'"'));
+            if (aliasParts.length === 1) continue;
```

Comment:
Why aren't we happy with a length of one? Is this a way of saying that there's no commas (i.e only `ID` is specified)?

Reply:
If we only get `[ID]` then it's better to not make an alias, as if we don't have any aliases then we can use a max limit of 100000. As soon as we make an alias, this limit goes down to 10000, so this is just a small optimization. 

Added a comment to the code to explain this.